src/export_workers.ts

```diff
@@ -0,0 +1,39 @@
+import createApi from './api'
+import { Worker, WorkerId } from '@joystream/types/working-group'
+
+main()
+
+async function main() {
+  const api = await createApi()
+
+  const result = {
+    storage: await getAllWorkers(api.query.storageWorkingGroup),
+    gateway: await getAllWorkers(api.query.gatewayWorkingGroup),
+  }
+  console.log(JSON.stringify(result))
+
+  api.disconnect()
+}
+
+interface WorkingGroup {
+  nextWorkerId: () => Promise<WorkerId>
+  workerById: (id: number) => Promise<Worker>
+}
+
+async function getAllWorkers(workingGroupApi: WorkingGroup): Promise<any[]> {
+  const next = ((await workingGroupApi.nextWorkerId()) as WorkerId).toNumber()
+
+  const workers = []
```

Comment:
When workers leave their role, they are removed from the map. So iteration from 0 to next worker id will produce null workers. Instead use the `.keys()` or `.entries()` to get only the existing workers in the map.

Example usage: https://github.com/Joystream/joystream/blob/c5a8bbaf4e4aa19de435278c22fd3530bacedfc9/storage-node/packages/runtime-api/workers.js#L123

Reply:
Done.